dask/array/core.py

```diff
@@ -759,16 +759,16 @@ def map_blocks(
     the example below, ``func`` will result in an ``IndexError`` when computing
     ``meta``:
 
-    >>> da.map_blocks(lambda x: x[2], da.random.random(5), meta=np.array(()))
+    >>> da.map_blocks(lambda x: x[2], da.random.default_rng().random(5), meta=np.array(()))
```

Comment:
```suggestion
    >>> rng = da.random.default_rng()
    >>> da.map_blocks(lambda x: x[2], rng.random(5), meta=np.array(()))
```
Not a blocker. Just personally find it clearer to define the generator separately in examples.